bin/build.sh

```diff
@@ -5,46 +5,159 @@ set -euo pipefail
 cd "$(dirname "${BASH_SOURCE[0]}")/.."
 . bin/stack-helpers.sh
 
```

Comment:
We'll also need to update the instructions at:
https://github.com/heroku/base-images/blob/main/BUILD.md

...since they reference the single architecture `installed-packages.txt`, list the image tag names, talk about variants (of which we now have the arch variants) etc.

I'm happy for that to be done in a second PR though if preferred.

Reply:
Filed #250